api/src/controllers/AggregationProcessorController-test.js

```diff
@@ -0,0 +1,29 @@
+import { expect } from 'chai';
```

Comment:
I'm sure this probably doesn't match what we've done elsewhere in the controllers folder, but I think we should make a aggregationProcessor subfolder with these tests and the request handlers contained within that.